packages/contracts-bedrock/test/setup/Setup.sol

```diff
@@ -95,6 +99,8 @@ contract Setup {
         deploy.run();
 
         optimismPortal = OptimismPortal(deploy.mustGetAddress("OptimismPortalProxy"));
+        optimismPortal2 = OptimismPortal2(deploy.mustGetAddress("OptimismPortalProxy"));
```

Comment:
missing label for portal 2? not that big of a deal

Reply:
@smartcontracts Configured feature flagged such that the legacy portal is initialized in the proxy if `useFaultProofs` is false, otherwise it uses the new portal.